skimage/measure/tests/test_structural_similarity.py

```diff
@@ -1,11 +1,21 @@
+import os
 import numpy as np
-from numpy.testing import assert_equal, assert_raises
+import scipy.io
+from numpy.testing import (assert_equal, assert_raises, assert_almost_equal,
+                           assert_array_almost_equal)
 
 from skimage.measure import structural_similarity as ssim
-
+from skimage.measure._structural_similarity import (gaussian_filter2,
+                                                    _discard_edges)
+import skimage.data
+from skimage.io import imread
+from skimage import data_dir
 
 np.random.seed(1234)
 
+cam = skimage.data.camera()
+cam_noisy = skimage.data.load('camera_noisy.png')
```

Comment:
If possible, it would be best to load the usual cameraman for testing purposes and generate a pseudorandom noisy version with `skimage.util.random_noise` with the appropriate settings and a set `seed`. 

To keep the package size down, we may need a rebase which omits the original commit of the noisy cameraman.